.ci/scripts/shellcheck

```diff
@@ -1,3 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This might be required, since the `pre-commit` will use it

https://github.com/elastic/apm-integration-testing/blob/da2f61a7b29131b6a436f6eb8bd437878f67672b/.pre-commit-config.yaml#L18-L25

Reply:
That `pre-commit` hook is external, how does it reference this script?

Reply:
IIRC, the `PATH` is set to point to `.ci/scripts` so it has a higher precedency. For vary reasons:

* We didn't want to depend on installing tools in the worker
* Avoid any security issues by fixing what version to be used since those are docker images we manage.
* Avoid to installing tools during the CI build if possible.

https://github.com/elastic/apm-integration-testing/blob/da2f61a7b29131b6a436f6eb8bd437878f67672b/.ci/Jenkinsfile#L91 is calling `preCommit` and https://github.com/elastic/apm-pipeline-library/blob/4bf7553413212e90a142c649ebb2ebac7b4a5503/vars/preCommit.groovy#L70-L75 is the 'magic' to reset the PATH env variable and use those tools.


A bit opinionated and likely hidden though

Reply:
I see, thanks! I'll replace the file and add a comment. I only removed it because I thought it was dead code.